src/api/cubrid_log.c

```diff
@@ -467,28 +676,34 @@ cubrid_log_send_configurations (void)
   if (css_send_request_with_data_buffer
       (g_conn_entry, NET_SERVER_CDC_START_SESSION, &rid, request, request_size, reply, reply_size) != NO_ERRORS)
     {
-      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, trace_errbuf);
+      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT,
+				 "Request(NET_SERVER_CDC_START_SESSION) failed. request size (%d), reply_sze (%d)\n",
+				 request_size, reply_size);
     }
 
   if (css_receive_data (g_conn_entry, rid, &recv_data, &recv_data_size, g_connection_timeout * 1000) != NO_ERRORS)
     {
-      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, trace_errbuf);
+      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT,
+				 "receive data from the request(NET_SERVER_CDC_START_SESSION) failed. (timeout : %d sec)\n",
+				 g_connection_timeout);
     }
 
   if (recv_data == NULL || recv_data_size != sizeof (int))
     {
-      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, trace_errbuf);
+      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, "recv_data is %s, receive data size : %d (should be %d)\n",
+				 recv_data ? "not null" : "null", recv_data_size, sizeof (int));
     }
 
   or_unpack_int (recv_data, &reply_code);
 
   if (reply_code == ER_CDC_NOT_AVAILABLE)
     {
-      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_UNAVAILABLE_CDC_SERVER, trace_errbuf);
+      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_UNAVAILABLE_CDC_SERVER,
+				 "Failed to connect to the server. Please check 'supplemental_log' parameter\n");
     }
   else if (reply_code != NO_ERROR)
     {
-      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, trace_errbuf);
+      CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, NULL);
```

Comment:
Is it possible to set it to NULL? This means the followings:

In the cubrid_log_tracelog() function.
- va_start (arg_ptr, **NULL**);
- vfprintf (g_trace_log, **NULL**, arg_ptr);

Reply:
No, 
Second argument for CUBRID_LOG_ERROR_HANDLING indicates 'msg'. 
and if it is NULL, it just set the error code and goto error label (cubrid_log_error:) 

However, it needs to be set some error message for more clear information about why "CUBRID_LOG_FAILED_CONNECT" error is returned.

Reply:
Please leave additional information for us to find the problem easily when the error has occurred in the customer environment.